### PR TITLE
Update dataController

### PR DESCRIPTION
**Pull request by jxm0428**
Update dataController: addProvider, selectProvider, selectService
Add isMatch(String)

### CODE REVIEW

File: src/dataController.java [hunk 1]
```diff
@@ -228,4 +228,32 @@ void deleteMember (memberData obj)
...
+
+    //returns a provider inside of dataController from user input
+    providerData selectProvider(String providerName) {
+        providerData find = new providerData();
+        for (providerData provider : providers) {
+            if (provider.isMatch(providerName) == true) {
+                find = provider;
+            }
+        }
+        return find;
```
Comment (romanbog) [inline src/dataController.java:245]: What happens when there is no match to a provider?

Comment (jxm0428) [reply src/dataController.java:245]: I will work on it!

Comment (romanbog) [reply src/dataController.java:245]: Consider changing the input from string to a scanner, and putting the function inside of an Interface.start() option to test it.
You could force the user to keep putting in input until a successful match is found, and then return the match.

**Review by romanbog — CHANGES_REQUESTED** [review]
revise dataController.selectProvider() and dataController.selectService

**Comment by jxm0428** [comment]
Got it! Thank you!

**Pull request merged**